src/Illuminate/Translation/Translator.php

```diff
@@ -216,7 +220,7 @@ public function trans($id, array $parameters = [], $domain = 'messages', $locale
      * Get a translation according to an integer value.
      *
      * @param  string  $id
-     * @param  int     $number
+     * @param  int|array|\Countable  $number
```

Comment:
@GrahamCampbell what contract do you mean? There is no contracts for translator. Only Symfony TranslatorInterface